proposalservice/src/main/java/ua/nure/proposalservice/controller/ApiException.java

```diff
@@ -0,0 +1,25 @@
+package ua.nure.proposalservice.controller;
```

Comment:
This class should be located not in controller, but in the exception or dto package